src/test/java/Homework16.java

```diff
@@ -0,0 +1,33 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.time.Duration;
+
+public class Homework16 extends BaseTest {
+
+    @Test
+    public void RegistrationNavigation() {
+        ChromeOptions options = new ChromeOptions();
+        options.addArguments("--remote-allow-origins=*"); // resolve issue with web socket error
+        WebDriver driver = new ChromeDriver(options);
+
+        driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
+
+        String url = "https://qa.koel.app/";
+        driver.get(url); //open page
+
+        WebElement registrationBtn = driver.findElement(By.cssSelector("[href='registration']"));
+        registrationBtn.click();
+
+
+        WebElement submitBtn = driver.findElement(By.cssSelector("[id='button']"));
+
+        Assert.assertFalse(submitBtn.isDisplayed());
```

Comment:
You got this problem, please correct it. Please finish your homework.

![image](https://github.com/AndreyZlotnikov/A50-QA-Automation-Framework/assets/104761482/7995ae66-8c41-40fc-acc8-aa40f527b212)

HINT

![image](https://github.com/AndreyZlotnikov/A50-QA-Automation-Framework/assets/104761482/69132d86-5d44-4b90-887f-54fe2b9c5797)